### PR TITLE
Release 0.1.0

### PR DESCRIPTION
**Pull request by jorgeramirezamora**
## Changes

- Add scripts and updates docs for publishing the `presto-js-client` in a private local registry and NPM.
- Migrate to `CommonJS` instead of `ESModule` to allow a smoother integration with the community.
- Use `@commitlint/config-conventional` instead of `@commitlint/config-angular`

## Notes

Git tags:
https://github.com/prestodb/presto-js-client/tags
GitHub releases:
https://github.com/prestodb/presto-js-client/releases
NPM package:
https://www.npmjs.com/package/@prestodb/presto-js-client/v/0.1.0

## Screenshots

Package published in private local registry:
image

Package published in NPM:
image

### CODE REVIEW

File: presto-client/.swcrc [hunk 1]
```diff
@@ -1,29 +1,29 @@
 {
+  "exclude": [
```
Comment (jorgeramirezamora) [inline presto-client/.swcrc:2]: The changes in this file are just sorting

File: presto-client/.swcrc [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
       "legacyDecorator": true
-    },
-    "keepClassNames": true,
-    "externalHelpers": true,
-    "loose": true
+    }
   },
   "module": {
-    "type": "es6"
+    "type": "commonjs"
```
Comment (jorgeramirezamora) [inline presto-client/.swcrc:26]: This is the only change here

**Review by tdcmeehan — APPROVED** [review]

**Review by alonsovb — APPROVED** [review]

File: presto-client/README.md [hunk 3]
```diff
@@ -4,12 +4,103 @@ This library was generated with [Nx](https://nx.dev).
...
+
+4. If the output of the previous command is successful, check that:
+
+- A new local git tag was generated.
+- `presto-client/CHANGELOG.md` file was updated.
+- `presto-client/package.json` file was updated.
+
+5. Visit http://localhost:4873 and ensure that `@prestodb/presto-js-client` was pushed successfully to the private local registry, including the git tag, `CHANGELOG.md`, and `package.json` files reviewed above.
+
+6. Now you can run `npm i @prestodb/presto-js-client` in any other project locally to test the released version before releasing it to NPM.
```
Comment (yhwang) [inline presto-client/README.md:63]: It would be nice to have a script to verify this and a npm run. But it's good for now!

**Review by yhwang — APPROVED** [review]
/lgtm

File: presto-client/README.md (hunk 3 above)
Comment (jorgeramirezamora) [reply presto-client/README.md:63]: Yeah def this can be improved. Thanks for the review 🥇

**Pull request merged**